standard-integration/server/ruby/server.rb

```diff
@@ -0,0 +1,70 @@
+require 'dotenv'
+require 'paypal_server_sdk'
+require 'sinatra'
+require 'sinatra/json'
+
+include PaypalServerSdk
+
+Dotenv.load(".env")
+
+set :port, 8080
+
+paypal_client = PaypalServerSdk::Client.new(
+  client_credentials_auth_credentials: ClientCredentialsAuthCredentials.new(
+    o_auth_client_id: ENV['PAYPAL_CLIENT_ID'],
+    o_auth_client_secret: ENV['PAYPAL_CLIENT_SECRET']
+  ),
+    environment: Environment::SANDBOX,
+    logging_configuration: LoggingConfiguration.new(
+      mask_sensitive_headers: false,
+      log_level: Logger::INFO,
+      request_logging_config: RequestLoggingConfiguration.new(
+        log_headers: true,
+        log_body: true,
+        ),
+      response_logging_config: ResponseLoggingConfiguration.new(
+        log_headers: true,
+        log_body: true
+      )
+  )
+)
+
+# Health Check
+get '/' do
+  json :message => "Server is running"
+end
+
+# Create an order to start the transaction.
+#
+# @see https://developer.paypal.com/docs/api/orders/v2/#orders_create
+post "/api/orders" do
+  # use the cart information passed from the front-end to calculate the order amount detals
+  cart = JSON.parse request.body.read
+  order_response = paypal_client.orders.orders_create({
+    'body' => OrderRequest.new(
+      intent: CheckoutPaymentIntent::CAPTURE,
+      purchase_units: [
+        PurchaseUnitRequest.new(
+          amount: AmountWithBreakdown.new(
+            currency_code: 'USD',
+            value: '100.00'
+          )
+        )
+      ]
+    ),
+    'prefer' => 'return=representation'
+    })
+  order_response.data.to_json
```

Comment:
@LeonarthusMectus can you return this response as json, similar to Line 35. By default `text/html;charset=utf-8` will be return instead

Reply:
@amsourav Good catch, fixed this up in both standard & advanced integrations.